CODEOWNERS

```
@@ -1,5 +1,23 @@
-# Code in the `rasa.shared` package is potentially re-used by downstream dependencies
-# such as Rasa X. Hence, changes within this package require double checking.
-/rasa/shared/ @RasaHQ/enable-squad
-/docs/docs/prototype-an-assistant.mdx @ricwo @alwx
+/docs/docs/prototype-an-assistant.mdx @RasaHQ/enable-squad
 /.github/workflows/ @RasaHQ/infrastructure
+/rasa/ @RasaHQ/enable-squad
+/rasa/core/featurizers/ @RasaHQ/research
+/rasa/core/policies/*.py @RasaHQ/research
```

Should we do a double star to match also subdirectories in case they're created?
```suggestion
/rasa/core/policies/**.py @RasaHQ/research
```

---

or we just do 
```suggestion
/rasa/core/policies/ @RasaHQ/research
```

---

Yup, the latter approach makes more sense 👍